README.md

```diff
@@ -51,19 +51,20 @@ Usage
 
 `cibuildwheel` runs inside a CI service. Supported platforms depend on which service you're using:
 
-|                 | Linux | macOS | Windows | Linux ARM | macOS ARM |
-|-----------------|-------|-------|---------|-----------|-----------|
-| GitHub Actions  | ✅    | ✅    | ✅      | ✅¹       | ✅²       |
-| Azure Pipelines | ✅    | ✅    | ✅      |           | ✅²       |
-| Travis CI       | ✅    |       | ✅      | ✅        |           |
-| AppVeyor        | ✅    | ✅    | ✅      |           | ✅²       |
-| CircleCI        | ✅    | ✅    |         |           | ✅²       |
-| Gitlab CI       | ✅    |       |         |           |           |
-| Cirrus CI       | ✅    | ✅³   | ✅      | ✅        | ✅        |
+|                 | Linux | macOS | Windows | Linux ARM | macOS ARM | Windows ARM |
+|-----------------|-------|-------|---------|-----------|-----------|-------------|
+| GitHub Actions  | ✅    | ✅    | ✅      | ✅¹       | ✅²       | ✅⁴         |
+| Azure Pipelines | ✅    | ✅    | ✅      |           | ✅²       | ✅⁴         |
```

Comment:
Why is it only supported by these two services? Availability of compiler toolchains perhaps?

Reply:
I haven't tested the others, but Cirrus is definitely missing the ARM64 toolchain.

Given the general lack of availability, I'd be very surprised if Travis or Appveyor have added the packages (unless they've done a "Select All", in which case they've got a 200GB or so image for Windows, which would be just as surprising to me)

Reply:
According to https://docs.travis-ci.com/user/reference/windows/ Travis doesn't have the packages (it uses the `visualstudio2017-workload-vctools` chocolatey package, which is too old for this)

Appveyor, apparently, does include the toolset https://github.com/appveyor/build-images/blob/master/scripts/Windows/install_vs2022.ps1#L206

So I'll mark Appveyor as supported.

Reply:
👍